src/Components/DataHistoryExplorer/DataHistoryExplorer.tsx

```diff
@@ -488,7 +488,11 @@ const DataHistoryExplorer: React.FC<IDataHistoryExplorerProps> = (props) => {
     useEffect(() => {
         if (data && !isLoading) {
             const seriesIdsWithNoData = state.timeSeriesTwins
-                ?.filter((ts) => !data?.find((d) => d.seriesId === ts.seriesId))
+                ?.filter(
```

Comment:
it is for the scenario I mentioned below, we need to make sure that the series has a property selected even if the data does not include that series id. Otherwise when you first open the modal from property inspector - the autofilled series with twin id would have a no data triangle although the property hasn't been selected yet:

it happens when you pass an series initially without an property selected. This is the scenario when you click on the data history control icon in property inspector - we pass the twin id automatically but user still needs to select the property to choose:
![image](https://user-images.githubusercontent.com/45217314/218567782-d76d77ef-21b3-4459-b66b-aec65716aea8.png)